### PR TITLE
RAC-379: Rework SummaryTable in React

### PR DESCRIPTION
**Pull request by tseho**
**Description (for Contributor and Core Developer)**

- [x] SummaryTable
- [x] SummaryDetails table
- [x] Split in dedicated files
- [ ] Warnings (TODO: check with Stephane the Link color, ~~is the WARNING label still necessary?~~ seen with Felix & Steph we remove it)
- [x] Test with Failures
- [x] unit tests

image

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by StevenVAIDIE — CHANGES_REQUESTED** [review]
NOT GTM !

**Review by SamirBoulil — APPROVED** [review]

**Comment by SamirBoulil** [comment]
Don't forget to show the design to stephane before merging

**Comment by ValentinMumble** [comment]
> Don't forget to show the design to stephane before merging

@SamirBoulil never!!!!!

**Comment by SamirBoulil** [comment]
ok

**Review by SamirBoulil — CHANGES_REQUESTED** [review]
👏🏼

**Review by juliensnz — APPROVED** [review]

**Review by SamirBoulil — APPROVED** [review]

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/job/execution/summary/SummaryTable.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            ))}
+            {step.failures.map((failure, index) => (
+              <Table.Row key={index}>
+                <LargeCell>
+                  <Helper level="error">{'string' === typeof failure ? failure : failure.label}</Helper>
+                </LargeCell>
+              </Table.Row>
+            ))}
+          </React.Fragment>
+        ))}
```
Comment (StevenVAIDIE) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/job/execution/summary/SummaryTable.tsx:99]: It miss the jobExecution failures loop : https://github.com/akeneo/pim-community-dev/pull/13607/files#diff-9bddd3cdf14e851a1e334fe083fa1669336877d01bd9c11dc4c7a03e5d441c7eL110

**Review by StevenVAIDIE — APPROVED** [review]

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/job/execution/summary/SummaryTable.tsx (hunk 1 above)
Comment (ValentinMumble) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/job/execution/summary/SummaryTable.tsx:99]: Added here https://github.com/akeneo/pim-community-dev/pull/13607/commits/fae6e154a1ca837323857ba9a3bf6a54194c32be

**Pull request merged**